plugins/VersionUpgrade/VersionUpgrade44to45/VersionUpgrade44to45.py

```diff
@@ -16,6 +26,79 @@
 }
 
 class VersionUpgrade44to45(VersionUpgrade):
+    def __init__(self) -> None:
+        """
```

Comment:
Thanks for using docstrings :)

Reply:
I'm used to using that for [personal projects](https://github.com/Ghostkeeper/Luna/blob/master/luna/plugins.py). Currently it looks a bit weird with the combination of styles though.